wasm2c/wasm-rt.h

```diff
@@ -326,6 +354,31 @@ typedef struct {
   bool is64;
 } wasm_rt_memory_t;
 
+#ifdef WASM_RT_C11_AVAILABLE
+/** A shared Memory object. */
+typedef struct {
+  /** The linear memory data, with a byte length of `size`. The memory is marked
```

Comment:
Newline after `/**` for multiline comments (at least thats the style I assume we use here).

Reply:
Happy to make this change, but the wasm-rt.h header doesn't seem to have much consistency on this. Again a broader cleanup maybe needed as a part of a separate PR.

Reply:
Sure thing.  Happy to see cleanup here or as separate PRs either before or after this lands.